### PR TITLE
Add link to new workload set documentation

### PR DESCRIPTION
**Pull request by tdykstra**
The markdownlint errors were already in this text before this PR.

### CODE REVIEW

File: release-notes/9.0/preview/preview7/sdk.md [hunk 1]
```diff
@@ -47,24 +47,24 @@ You can see what mode your SDK installation is in by running `dotnet workload --
...
-   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.0.2\WorkloadManifest.json
-   Install Type:              Msi
+   Installation Source: SDK 8.0.400
+   Manifest Version:    8.1.0/8.0.100
+   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.1.0\WorkloadManifest.json
+   Install Type:        FileBased
 ```
 
-In this example, I am in 'manifest' mode, which is what we call the current mode of managing workloads.
+In this example, I am in 'manifests' mode, which is what we call the current mode of managing workloads.
```
Comment (mairaw) [inline release-notes/9.0/preview/preview7/sdk.md:59]: not your text but maybe we should change the text here to not use I?

```suggestion
This example uses the 'manifests' mode, which is what we call the current mode of managing workloads.
```

File: release-notes/9.0/preview/preview7/sdk.md [hunk 2]
```diff
@@ -47,24 +47,24 @@ You can see what mode your SDK installation is in by running `dotnet workload --
...
 > dotnet workload --info
-Workload version: 9.0.100-manifests.400dd185
+ Workload version: 9.0.100-preview.7.24414.1
 Configured to use loose manifests when installing new manifests.
  [aspire]
-   Installation Source: VS 17.10.35027.167, VS 17.11.35111.106
-   Manifest Version:    8.0.2/8.0.100
-   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.0.2\WorkloadManifest.json
-   Install Type:              Msi
+   Installation Source: SDK 8.0.400
```
Comment (mairaw) [inline release-notes/9.0/preview/preview7/sdk.md:53]: is this a valid result for .NET 9 preview 7?

Comment (baronfel) [reply release-notes/9.0/preview/preview7/sdk.md:53]: If the aspire workload was last installed when using 8.0.400-band SDKs, then yes - the next time a workload update happened (via `dotnet workload install` or `dotnet workload update`) I'd expect this to change to align with the in-use SDK's feature band.

**Comment by tdykstra** [comment]
@rbhanda @leecow @baronfel 
I fixed the preexisting Markdownlint errors, so this is ready to be merged now.

**Review by leecow — APPROVED** [review]
Looks good.

**Pull request merged**